psi4/driver/schema_wrapper.py

```diff
@@ -541,14 +541,8 @@ def run_json_qcschema(json_data, clean, json_serialization, keep_wfn=False):
 
     # Handle special properties case
     if json_data["driver"] == "properties":
-        if "properties" in json_data["model"]:
-            kwargs["properties"] = [x.lower() for x in json_data["model"]["properties"]]
-
-            extra = set(kwargs["properties"]) - can_do_properties_
-            if len(extra):
-                raise KeyError("Did not understand property key %s." % kwargs["properties"])
-        else:
-            kwargs["properties"] = list(can_do_properties_)
+        if not "properties" in kwargs:
+            kwargs["properties"] = list(default_properties_)
```

Comment:
I guess we won't catch properties like GRID_ESP that can't be returned, but as this is mostly for CC, perhaps that's ok.

Reply:
yes, there are some edge cases that will probably slip through. maybe we could collect them and patch them through in another way, if anyone is interested in the future.